Tests/Swift Tests/BugsnagSwiftPublicAPITests.swift

```diff
@@ -0,0 +1,368 @@
+//
+//  BugsnagSwiftPublicAPITests.swift
+//  Tests
+//
+//  Created by Robin Macharg on 15/05/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import XCTest
+
+/**
+ * Test all public APIs from Swift.  Purely existence tests, no attempt to verify correctness
+ */
+
+class FakePlugin: NSObject, BugsnagPlugin {
+    func load(_ client: BugsnagClient!) {}
```

Comment:
Should this be `BugsnagClient` instead?

Reply:
This is what was offered by XCode's autofix.  I've updated the protocol definitiion to be explicitly `_Nonnull` for the client arg which fixes things.